packages/components/src/components/bal-list/bal-list.scss

```diff
@@ -394,7 +443,13 @@ bal-card > bal-list.p-0 {
   }
 
   &.is-open bal-icon.trigger-icon {
```

Comment:
the bal-icon has a turn prop which changes the rotation. Could you use that instead?